README.md

```diff
@@ -40,5 +40,12 @@ To run the local server, run:
 $ yarn dev
 ```
 
+To run locally but connected to the testnet:
+
+```
+$ yarn build
```

Comment:
yarn build is only necesary to run before deploying a new version, just yarn start is enough for local development

Reply:
soled